languages/csharp/reference/README.md

```diff
@@ -161,24 +164,20 @@ This report is updated automatically when any _design.md_ document is modified o
 
 ### Unallocated Concepts
 
-- bitwise-manipulation _(unallocated-concepts)_
 - conditionals-do-while _(unallocated-concepts)_
```

Comment:
@mikedamay I think we have a `while-loops` concept introduced in some exercise.

Reply:
Yep, `floating-point-numbers` but I need to add a little text about `do-while`.